### PR TITLE
Pin dependencies to minor version

### PR DESCRIPTION
**Pull request by Hkly**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by DevShep** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -22,31 +22,31 @@
...
-    "karma-phantomjs-launcher": "^1.0.0",
+    "browserify": "12.0.x",
+    "chai": "3.5.x",
+    "chai-as-promised": "5.2.x",
+    "jsgreat": "0.1.x",
+    "karma": "0.13.x",
+    "karma-browserify": "5.0.x",
+    "karma-mocha": "0.2.x",
+    "karma-phantomjs-launcher": "1.0.x",
     "karma-spec-reporter": "0.0.24",
```
Comment (LionOps) [inline package.json]: Why not pin this too?

Comment (Hkly) [inline package.json]: I assumed we were pinning it at `0.0.24` for a reason

**Comment by LionOps** [comment]
~

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by LionOps** [comment]
:+1:

**Pull request merged**